.github/workflows/unit-test.yml

```
@@ -62,8 +62,9 @@ jobs:
           kubectl: 1.18.4
           kubeval: v0.16.1
           command: |
-            kubeval --strict ${{ env.K8S_MANIFEST_DIR }}/k8s/*.yml
-            kubeval --strict ${{ env.K8S_MANIFEST_DIR }}/k8s/kustomize/base/cloudbeat*.yml
+            kubeval --strict ${{ env.K8S_MANIFEST_DIR }}/k8s/**/*.yml --ignore-missing-schemas
+            kubeval --strict ${{ env.K8S_MANIFEST_DIR }}/eks/*.yml --ignore-missing-schemas
+            kubeval --strict ${{ env.K8S_MANIFEST_DIR }}/kustomize/base/*.yml --ignore-missing-schemas
```

maybe we can use `directories` flag instead of glob matching here
```suggestion
            kubeval --strict --directories ${{ env.K8S_MANIFEST_DIR }} --ignore-missing-schemas --ignored-filename-patterns kustomization
```

---

That is a nice idea, but do we really want to test the entire directory?

Adv
- Maintenance - totally agree
- Can prevent future bugs

Dsv
- There are some files that I think should not be validated by the `kubeval` (the overlays files for example) - these files will always produce warnings, maybe they can even produce errors and still works. I just the solution is a bit too generalized.

That being said, I don't think it's so critical - if you think it's the best way after reading what I wrote - NP let me know and I'll change it.

---

yes I would go with what I suggested since it will also prevent us from breaking the overlays. I don't mind the warning as I explained above.